### PR TITLE
Added Dance-Form-Classification project

### PR DESCRIPTION
**Pull request by stiwari-ds**
# Pull Request for DL-Simplified 💡

## Issue Title :  Dance Form Classification

- **Info about the related issue (Aim of the project)** :  Training a deep learning model to classify eight Indian classical dance forms with a limited number of images.
- **Name:** Siddhant Tiwari
- **GitHub ID:** stiwari-ds
- **Email ID:** siddhant.tiwari.ds@gmail.com
- **Idenitfy yourself: (Mention in which program you are contributing in. Eg. For a JWOC 2022 participant it's, `JWOC Participant`)** SWOC Participant

Closes: #73

### Describe the add-ons or changes you've made 📃

Added deep learning models in an incremental manner starting with basic Convolutional Neural Networks from scratch up to transfer learning techniques of feature extraction and fine-tuning.   
Able to improve performance (accuracy) from 41% to 83% on the test set, with significant improvements for every technique added into the process.

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️
Results tested over multiple runs of the jupyter (Google Colab) notebooks. (Requires GPU)

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by 072arushi — APPROVED** [review]
good contribution

**Comment by abhisheks008** [comment]
Approved!
@stiwari-ds
